simpleLooping/test_removeSpaceAtBeginning.py

```diff
@@ -0,0 +1,15 @@
+import unittest
+from removeSpaceAtBeginning import removeSpaceAtBeginning
+
+class TestRemoveSpaceAtBeginning(unittest.TestCase):
+    def test_emptyList(self):
+        self.assertEqual(removeSpaceAtBeginning(''), '')
```

Comment:
Empty string should be using “”, not ‘’. Only single character using single quote notation.

Reply:
Please review this.